### PR TITLE
Make GetUp/Down/Left/Right pure virtual in Track

### PR DESCRIPTION
**Pull request by dimitry-**
This will let us move implementation around with more confidence.

Test: builds

### CODE REVIEW

File: src/OrbitGl/TimerTrack.cpp [hunk 1]
```diff
@@ -339,6 +339,14 @@ std::string TimerTrack::GetTooltip() const {
          "functions";
 }
 
+const TimerInfo* TimerTrack::GetLeft(const TimerInfo& timer_info) const {
+  return timer_data_->GetFirstBeforeStartTime(timer_info.start(), timer_info.depth());
+}
+
+const TimerInfo* TimerTrack::GetRight(const TimerInfo& timer_info) const {
+  return timer_data_->GetFirstAfterStartTime(timer_info.start(), timer_info.depth());
+}
```
Comment (dimitry-) [inline src/OrbitGl/TimerTrack.cpp:348]: These were not implemented - they might be unused but I will address this in the followup pr

File: src/OrbitGl/BasicPageFaultsTrack.cpp [hunk 2]
```diff
@@ -106,4 +106,24 @@ float BasicPageFaultsTrack::GetAnnotatedTrackContentHeight() const {
...
+const orbit_client_protos::TimerInfo* BasicPageFaultsTrack::GetUp(
+    const orbit_client_protos::TimerInfo& /*info*/) const {
+  return nullptr;
+}
+
+const orbit_client_protos::TimerInfo* BasicPageFaultsTrack::GetDown(
+    const orbit_client_protos::TimerInfo& /*info*/) const {
+  return nullptr;
+}
+
```
Comment (vickyliu-go4it) [inline src/OrbitGl/BasicPageFaultsTrack.cpp:128]: In `GraphTrack` we already mention that these are not supported and will always return nullptr. Do we still need to override these methods in the derived classes?

Comment (dimitry-) [reply src/OrbitGl/BasicPageFaultsTrack.cpp:128]: right - removed.

**Review by vickyliu-go4it — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**